firmware/views.py

```diff
@@ -0,0 +1,178 @@
+# coding: utf8
+""" Routes for app paths and functions combining multiple modules """
+
+
+import sqlalchemy.orm
+from flask import request, session, redirect, url_for, render_template, flash
+from firmware import app
+from firmware import repository
+from firmware.models import Company, User, Review
+from firmware.uploaders import upload_file
+from firmware.forms import AddUser, AddCompany, LogIn, AddReview
+from firmware.decorators import login_required
+from firmware.authorization import authorize_edit_company
+
+
+
+def get_details(company_id):
+    """get details about a company via it's id"""
+    try:
+        company = repository.get_company(company_id)
+    except sqlalchemy.orm.exc.NoResultFound:
+        errors = dict()
```

Comment:
You can rewrite these 2 lines like this: `errors = {'company-not-found']: "Company not found!" }`.